airbyte-integrations/connectors/source-github/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.3.9
```

Comment:
@girarda what should be here? I guess maybe 0.3.10 and we release a new version of this as well?

Reply:
yes, let's bump to 0.3.10

Reply:
Done, putting a reminder here that we can't publish til we confirm cdk 0.15 is available on pypi